app/services/doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -97,12 +97,19 @@ def processed_alerts
 
         def log_alerts(alerts)
           log_alert_results = {}
+          has_failed = {}
+
           alerts.keys.each do |key|
             alerts[key.to_sym].each do |alert|
-              side = alert[:side] || 'no_side'
-              log_alert_results[alert[:name].
+              alert_name_key = alert[:name].
                 downcase.
-                parameterize(separator: '_').to_sym] = { "#{side}": alert[:result] }
+                parameterize(separator: '_')
+              side = alert[:side] || 'no_side'
+              unless has_failed[alert_name_key.to_sym]
+                log_alert_results[alert_name_key.to_sym] =
+                  { "#{side}": alert[:result] }
+              end
+              has_failed[alert_name_key.to_sym] = !(key == :passed)
```

Comment:
So... given that we're processing the `:passed` first, I don't know that this actually buys us much. We would still have the problem of an overwrite, e.g. from `Failed` to `Attention`, but in practice, I just looked and couldn't find any transactions with multiple `Visible Pattern` alerts that weren't all `Failed`. I don't know if we could get some Failed and some Attention, so I don't know if it's worth changing to keep them in order of importance (Failed > Caution|Unknown > Attention > Passed).

I think we should either re-work this so Failed overwrites everything, Caution overwrites Attention, etc. OR cancel this PR and know that we processing passed first so they can be overwritten.

Reply:
We could either put a comma delimiter, between the errors.